src/descriptionPage/productDescription.ts

```diff
@@ -0,0 +1,169 @@
+import { productImg } from "../StorePage/products";
+
+const fragmentDescriptiontPage = document.createDocumentFragment();
+export const divDescriptiontPage = document.createElement("div");
+divDescriptiontPage.classList.add("description-page");
+fragmentDescriptiontPage.appendChild(divDescriptiontPage);
+
+const pathToProduct = document.createElement("div");
+pathToProduct.classList.add("path-to-product");
+divDescriptiontPage.appendChild(pathToProduct);
+
+const linkMain = document.createElement("a");
+linkMain.classList.add("description-page__links");
+linkMain.setAttribute("href", "");
+linkMain.textContent = "Store";
+pathToProduct.appendChild(linkMain);
+
+const p1 = document.createElement("p");
+p1.textContent = ">>";
+pathToProduct.appendChild(p1);
+
+const linkCategory = document.createElement("a");
+linkCategory.classList.add("description-page__links");
+pathToProduct.appendChild(linkCategory);
+
+const p2 = document.createElement("p");
+p2.textContent = ">>";
+pathToProduct.appendChild(p2);
+
+const linkBrand = document.createElement("a");
+linkBrand.classList.add("description-page__links");
+pathToProduct.appendChild(linkBrand);
+
+const p3 = document.createElement("p");
+p3.textContent = ">>";
+pathToProduct.appendChild(p3);
+
+const linkTitle = document.createElement("a");
+linkTitle.classList.add("description-page__links");
+pathToProduct.appendChild(linkTitle);
+
+const titleDescription = document.createElement("h2");
+titleDescription.classList.add("title-description");
+divDescriptiontPage.appendChild(titleDescription);
+
+const wrapDescription = document.createElement("div");
+wrapDescription.classList.add("wrap-description");
+divDescriptiontPage.appendChild(wrapDescription);
+
+export const wrapDescriptionImage = document.createElement("div");
+wrapDescriptionImage.classList.add("wrap-description__img");
+wrapDescription.appendChild(wrapDescriptionImage);
+
+const mainPhoto = document.createElement("img");
+mainPhoto.classList.add("main-photo");
+wrapDescriptionImage.appendChild(mainPhoto);
+
+export const wrapSlideImage = document.createElement("div");
+wrapSlideImage.classList.add("wrap-slide-image");
+wrapDescriptionImage.appendChild(wrapSlideImage);
+
+const wrapProductInfo = document.createElement("div");
+wrapProductInfo.classList.add("wrap-description__product-info");
+wrapDescription.appendChild(wrapProductInfo);
+
+const infoDescription = document.createElement("p");
+infoDescription.classList.add("product-info__p");
+wrapProductInfo.appendChild(infoDescription);
+
+const infoDiscont = document.createElement("p");
+infoDiscont.classList.add("product-info__p");
+wrapProductInfo.appendChild(infoDiscont);
+
+const infoRating = document.createElement("p");
+infoRating.classList.add("product-info__p");
+wrapProductInfo.appendChild(infoRating);
+
+const infoStock = document.createElement("p");
+infoStock.classList.add("product-info__p");
+wrapProductInfo.appendChild(infoStock);
```

Comment:
it's almost the same steps
Maybe it's better to create some kind of manager for this?